### PR TITLE
544 Revert change to setState in storage | use setItem on localstorag…

### PR DESCRIPTION
**Pull request by Winna95**
…e directly when setting the Role value

### CODE REVIEW

File: src/js/api/auth/login.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import { Store } from '../../storage/storage.js';
-import { apiPath } from '../constants.js';
```
Comment (Winna95) [inline src/js/api/auth/login.js:2]: unused variable, reported by eslint.

File: src/js/api/auth/login.js [hunk 2]
```diff
@@ -59,8 +58,9 @@ export async function login(profile) {
     const { token, role, email, id, ...filteredProfile } = await response.json();
 
     // The following code handles the log in response. part of this handling is to set the Role entry in our storage solution.
-    // before starting the handling we therefore clear the existing value for the Role entry. It will be set to the correct values depending on the current log in response.
-    new Store('Role', 'null', Boolean(remember !== 'on')).clear();
+    // before starting the handling we therefore clear the existing value for the Role entry.
+    // It will be set to the correct values depending on the current log in response.
+    localStorage.setItem('Role', 'null');
```
Comment (Winna95) [inline src/js/api/auth/login.js:63]: the 'Role'-entry is only read directly from localStorage. As such we don't need the indirection via the Store-class.

File: src/js/storage/storage.js [hunk 3]
```diff
@@ -21,10 +21,9 @@ export class Store {
 
   set state(state) {
     if (state) {
-      //Do not stringify value if value is already is a string ; to avoid adding "" around the string value
-      this.storage.setItem(this.key, typeof state === 'string' ? state : JSON.stringify(state));
```
Comment (Winna95) [inline src/js/storage/storage.js:25]: Revert of https://github.com/NoroffFEU/agency.noroff.dev/commit/86b2d794ee8fb4e4ad514c22f036abe383354ccd#diff-551b6093ecd9156be9fbb034108f4b1c4c0dda79477e186581e7b394abd6fddaR25 due to https://github.com/NoroffFEU/agency.noroff.dev/pull/544#discussion_r1446448390

**Review by maleneivy — APPROVED** [review]

**Comment by maleneivy** [comment]
I took the liberty to solve the merging conflict in storage.js. 
But there is a merge conflict with the auth/login.js

**Pull request merged**